ts/streaming/frontend/src/components/StreamOutExample.tsx

```diff
@@ -0,0 +1,44 @@
+import getRequestClient from "../lib/getRequestClient.ts";
+import { useState } from "react";
+import Button from "./Button.tsx";
+
+function StreamOutExample() {
+  const [logs, setLogs] = useState<string[]>([]);
+  const [isStreaming, setIsStreaming] = useState(false);
+
+  const startStream = async () => {
+    // Start the stream by listen for messages
+    const stream = await getRequestClient().streaming.logStream({ rows: 10 });
+    stream.socket.on("open", () => {
+      setLogs([]);
+      setIsStreaming(true);
+    });
+    stream.socket.on("close", () => setIsStreaming(false));
+
+    for await (const message of stream) {
+      setLogs((prev) => [...prev, message.message]);
+    }
+  };
+
+  return (
+    <div className="flex flex-col space-y-4">
+      <p>
+        This example is making use of <code>api.streamOut</code> to get a log
+        stream from the backend.
+      </p>
+
+      <Button disabled={isStreaming} onClick={startStream}>
+        Start upload stream
```

Comment:
```suggestion
        Start log stream
```
upload was copy/paste from the uploadStream I assume?

Reply:
Great, thanks!